api/src/controllers/users.js

```diff
@@ -83,7 +83,7 @@ const getInfoUserCtx = req => {
   }
 
   const roles = getRoles(req);
-  if (!auth.isOffline(roles)) {
+  if (auth.isOnlineOnly({ roles })) {
```

Comment:
I feel a little uncomfortable faking a userCtx here. It'll work perfectly right now but if we change the implementation to check something else on the userCtx this will regress. One solution would be to have another function which just takes the roles?

Reply:
Done.